tools/test_live_repair.sh

```diff
@@ -91,7 +91,7 @@ args+=( -t "127.0.0.1:8830" )
 gen=1
 # Initial seed for verify file
 if ! "$crucible_test" fill "${args[@]}" -q -g "$gen"\
-          --verify-out alan --retry-activate >> "$test_log" 2>&1 ; then
+          --verify-out alan >> "$test_log" 2>&1 ; then
```

Comment:
Is this change intentional?

Reply:
It is intentional yeah, though it's a drive by change and unrelated to this PR.
That option is not required for the live-repair test and was just copied
over from the previous test.  I noticed it when I copied `test_live_repair.sh` to make
`test_repair_special.sh`.  :)

There are probably a pile of things like this, and I debated if I should change
this here, or make a whole new PR with a bunch of clean up.  I then forgot
about it and it remained changed.

Reply:
Okay, I'm fine keeping it in this PR